src/modules/commander/commander_params.c

```diff
@@ -670,13 +670,14 @@ PARAM_DEFINE_INT32(NAV_RCL_ACT, 2);
 /**
  * RC loss exceptions
  *
- * Specify modes in which RC loss is ignored and the failsafe action not triggered.
+ * Specify modes in which the loss of non-essential stick input is ignored and no failsafe action is triggered.
```

Comment:
```suggestion
 * Specify modes in which the loss of stick input (manual control)  is ignored and no failsafe action is triggered.
```

Reply:
I feel the wording "non-essential" is rather confusing, and would instead add the keyword "manual control" to it, as it is used in many other places already to refer to "manual control stick inputs". Maybe that's the string we should go for? (To replace "RC input" consistently).